### PR TITLE
bpo-38495: Updated functions.rst: Clearer *sep* and *end* defaults explanation 

### PR DESCRIPTION
**Pull request by Seluj78**
https://bugs.python.org/issue38495

### CODE REVIEW

**Comment by Seluj78** [comment]
cc @vstinner

**Comment by Seluj78** [comment]
@bedevere-bot updated title for issue number

**Comment by Seluj78** [comment]
ping @JulienPalard for review

**Review by deronnax — APPROVED** [review]

**Comment by taleinat** [comment]
@Seluj78, I've seen you ping/CC specific core devs on several recent PRs. Please avoid doing so unless they've specifically asked you to.

Core devs can make sure they're notified on changes to specific files. If they haven't done so, they probably don't want to be actively pinged on every such change.

**Comment by taleinat** [comment]
That being said, @Seluj78, your enthusiasm and efforts to help improve Python are greatly appreciated! Please be patient with regard to our limited ability to address the many issues and PRs.

**Comment by Seluj78** [comment]
> @Seluj78, I've seen you ping/CC specific core devs on several recent PRs. Please avoid doing so unless they've specifically asked you to.

For Victor Stinner and Julien Palard, I know them very well (not an excuse, just a note) and they've told me that I could 😄

**Comment by taleinat** [comment]
Ah, @Seluj78, that's fine then 👍

**Comment by vstinner** [comment]
> For Victor Stinner and Julien Palard, I know them very well (not an excuse, just a note) and they've told me that I could smile

I confirm. But I only plan to quickly approve trivial doc fixes like typos, and let others review more complex ones :-) FYI Jules and Julien are actively working together on the french documentation translation, that's how they spot so many tiny doc issues ;-)

**Comment by Seluj78** [comment]
> FYI Jules and Julien are actively working together on the french documentation translation, that's how they spot so many tiny doc issues ;-)

We're not the only one ! Proudly introducing @deronnax and @awecx 🤗 ❤️

File: Doc/library/functions.rst [hunk 1]
```diff
@@ -1307,9 +1307,10 @@ are always available.  They are listed here in alphabetical order.
 
 .. function:: print(*objects, sep=' ', end='\\n', file=sys.stdout, flush=False)
 
-   Print *objects* to the text stream *file*, separated by *sep* and followed
-   by *end*.  *sep*, *end*, *file* and *flush*, if present, must be given as keyword
-   arguments.
+   Print *objects* to the text stream *file* (the standard output by default),
+   separated by *sep* (a space by default) and followed by *end*
+   (a newline by default).
+   *sep*, *end*, *file* and *flush*, if present, must be given as keyword arguments.
```
Comment (vstinner) [inline Doc/library/functions.rst:1313]: As a Python expert, I dislike such redundant documentation: the default value is already documented on the " .. function:: print(*objects, sep=' ', end='\\n', file=sys.stdout, flush=False)" line. Moreover, this line ("*objects, " part) also explains that these parameters are keyword-only (cannot be passed as positional arguments).

**Comment by vstinner** [comment]
> As a Python expert, ...

I mean that it's just my opinion. I have no idea if developers learning Python would prefer to have such "redundant" documentaiton. But it may be more expensive to maintain if we change the default values.

File: Doc/library/functions.rst (hunk 1 above)
Comment (Seluj78) [reply Doc/library/functions.rst:1313]: I understand where you're coming from, I personally feel it's clearer that way

Also I followed your advice when you told me to look into bugs.python.org for doc bugs and I did, it's on you 😉

**Comment by csabella** [comment]
I don't think this PR should be accepted.  The ticket on the bug tracker seemed more for the idea that the OP couldn't see the space in `sep` in Sphinx more than a need to explain the defaults more.  As Victor said, this is redundant and doesn't add much (if any) value.

-1 from me.

**Pull request closed without merging**